src/vaadin-text-field-mixin.html

```diff
@@ -117,8 +117,8 @@
 
   const HOST_PROPS = {
     default: ['list', 'autofocus', 'pattern', 'autocapitalize', 'autocorrect', 'maxlength',
-      'minlength', 'name', 'placeholder', 'autocomplete', 'title'],
-    accessible: ['disabled', 'readonly', 'required', 'invalid']
+      'minlength', 'name', 'placeholder', 'autocomplete', 'title', 'disabled', 'readonly', 'required'],
+    accessible: ['invalid']
```

Comment:
The `accessible` set is only a list of props that are also propagated to a corresponding `aria-*` attribute.

'disabled', 'readonly', 'required' were moved to the `default` set so they only propagate to the native HTML attribute of the same name on the input (without aria attribute).